Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -286,6 +292,9 @@ JSValue Stringifier::stringify(JSValue value)
     }
     if (UNLIKELY(stringifyResult != StringifySucceeded))
         return jsUndefined();
+#if FAST_STRINGIFY_LOG_USAGE
+    WTFLogAlways("Not fastStringify: %s", result.toString().utf8().data());
```

Comment:
JSC uses `dataLogLn("Not fastStringify: ", result.toString());`

Reply:
I am 100% willing to use `dataLogLn`; I will fix this right away.

Separately, I am deeply offended by the concept "JSC uses". Why should this be different in JavaScriptCore than in any other framework built on top of WTF? Is there some special requirement JavaScriptCore has that WebCore and WebKit do not?

Reply:
`dataLog` supports custom dumpers, which is why we use it.  dataLog gets printed to stderr by default.  For JSC folks, we work with the jsc shell, and we see the `dataLog` output.  For WebCore/WebKit, stderr is not always dumped to the terminal.  Hence, they rely on `WTFLogAlways` so that they can see the logging via syslog.  However, we now have a `JSC_useOSLog=1` option that will redirect everything printed via dataLog to syslog.  So, technically, WebCore and WebKit can use it too.

Reply:
OK, but still not happy with "technically". We should use one thing across the project. I am annoyed that we aren’t even trying to do that!